Core/Node-API/Source/js_native_api_v8_internals.h

```diff
@@ -85,4 +59,7 @@ class PersistentToLocal {
 #define CHECK_LE(a, b) CHECK((a) <= (b))
 #endif
 
+#define NAPI_PRIVATE_KEY(context)                                      \
+  (v8::Private::New(context->GetIsolate()))
```

Comment:
Did you look into the perf? Are we okay with the private key perf?

Reply:
But there was a perf regression right? So we have to do something.

Reply:
I pushed a fix: https://github.com/BabylonJS/JsRuntimeHost/pull/70/commits/2b67aedc2f8d487b7521c1e58cd8835d05049315. This makes the perf better than main now.